doc/index.rst

```diff
@@ -1,28 +1,48 @@
+Pylint documentation
+====================
 
-Pylint User Manual
-==================
+Pylint is a `static code analyser`_ for python 2 or 3. Only python interpreter above 3.7.2 are supported
+in the latest version.
```

Comment:
```suggestion
Pylint is a `static code analyser`_ for Python 3. The latest version supports Python 3.7.2 and newer.
```

Reply:
You can use pylint for python 2 : it's version 1.9.6 
```suggestion
Pylint is a `static code analyser`_ for python 2 or 3. Only python interpreter above 3.7.2 are still maintained currently.
```

Reply:
Not a super issue for me, but let's not mention Python 2 in the first line of our index. Small contribution to the general `Python` ecosystem to nudge everybody towards 3 😄

Reply:
I looked at mypy's doc for an example, this is why I changed it : https://mypy.readthedocs.io/en/stable/index.html

I don't think Guido can be suspected of trying to incite people to keep/start using python 2 😄

Reply:
I think the difference is that `mypy` can actually check `2.7` code although you need to run it using `3.6`. I don't actually know this: but do we still support checking `2.7` code? It feels like that would violate a lot of assumptions in our checkers which wouldn't work.

Reply:
We have old versions that do check python 2, some versions only python 2. If you pip install pylint you'll get a version compatible with your interpreter. I did not try to check if pylint 2.14 can check python 2 code, probably not as you say.

Reply:
```suggestion
Pylint is a `static code analyser`_ for Python 2 or 3. The latest version supports Python 3.7.2 and above.
```